src/Init/Prelude.lean

```diff
@@ -1121,47 +1121,26 @@ propositional connective is `Not : Prop → Prop`.
 export Bool (or and not)
 
 /--
-The type of natural numbers, starting at zero. It is defined as an
-inductive type freely generated by "zero is a natural number" and
-"the successor of a natural number is a natural number".
+The natural numbers, starting at zero.
 
-You can prove a theorem `P n` about `n : Nat` by `induction n`, which will
-expect a proof of the theorem for `P 0`, and a proof of `P (succ i)` assuming
-a proof of `P i`. The same method also works to define functions by recursion
-on natural numbers: induction and recursion are two expressions of the same
-operation from Lean's point of view.
-
-```
-open Nat
-example (n : Nat) : n < succ n := by
-  induction n with
-  | zero =>
-    show 0 < 1
-    decide
-  | succ i ih => -- ih : i < succ i
-    show succ i < succ (succ i)
-    exact Nat.succ_lt_succ ih
-```
```

Comment:
I can understand why this section was removed but my opinion is that it's worth having something around. The Nat type is probably most users' first inductive type, and maybe one of the first types at all, and one of the most natural candidates (no pun intended) for a new user's first proof (my first ever Lean proof was n + n = 2 * n), so giving an example of induction here is pretty valuable in my view.

Reply:
I think this belongs in the reference manual, where it's much easier to read long text. I'll make sure it's added there.

Reply:
https://github.com/leanprover/reference-manual/pull/367